src/RequestModel.php

```diff
@@ -15,14 +15,14 @@ public function setRequestData(array $requestData): void
         $this->requestData = $requestData;
     }
 
-    public function getValue(string $field, $default = null)
+    public function getAttributeValue(string $attribute, $default = null)
     {
-        return ArrayHelper::getValueByPath($this->requestData, $field, $default);
+        return ArrayHelper::getValueByPath($this->requestData, $attribute, $default);
     }
 
-    public function hasValue(string $field): bool
+    public function hasAttribute(string $attribute): bool
     {
-        return $this->getValue($field) !== null;
+        return $this->getAttributeValue($attribute) !== null;
```

Comment:
It's incorrect behaviour to check for the existence attribute via comparing value with `null`.
Please use [ArrayHelper::pathExists](https://github.com/yiisoft/arrays/blob/master/src/ArrayHelper.php#L958)

Reply:
fixed